### PR TITLE
Remove `MODALITIES` and `privbn` artifacts

### PR DESCRIPTION
**Pull request by fealho**
As part of Issue #215, this PR deletes:
- The `MODALITIES` attribute. We currently don't use it. If it is required for the `multi_table` and `time-series` cases, it should be added back when those classes are implemented.
- `privbn` artifacts which should've been removed in the PR #218.

### CODE REVIEW

File: sdgym/datasets.py [hunk 1]
```diff
@@ -122,11 +121,6 @@ def get_available_datasets():
 
 
 def _get_available_datasets(modality, bucket=None, aws_key=None, aws_secret=None):
-    if modality not in MODALITIES:
```
Comment (amontanez24) [inline sdgym/datasets.py:125]: Should we still have a check to make sure `modality` isn't some random string?

Comment (fealho) [reply sdgym/datasets.py:125]: Very good catch, I shouldn't have deleted `MODALITIES` in this file, it's unrelated to the one in `synthesizers/base.py`.

**Review by amontanez24 — APPROVED** [review]
Thanks for addressing!

**Review by pvk-developer — APPROVED** [review]
🚀 LGTM!

**Pull request merged**